### ISSUE TITLE
It seems not working for your example.

### ISSUE THREAD

**Issue by lunguyen1986** [rafaelklaessen/react-tradingview-widget#12]
Hi , 
I try to add a 'TradingViewWidget' component to root view of my application. But it always appear error. 'can't find variable: Themes'.

**Comment by rafaelklaessen** [comment]
Try
```javascript
Import TradingView, { Themes } from 'react-tradingview-widget`;
```

**Comment by lunguyen1986** [comment]
Yes, I imported this. But i don't work anymore

**Comment by rafaelklaessen** [comment]
Okay, so there was something wrong with the exporting of the constants.
I fixed in it v1.2.5, if you install it importing `Themes` should work.
